kbase-extension/static/kbase/js/common/ui.js

```diff
@@ -1193,7 +1193,7 @@ define([
                     fontSize = jsonBlockWidgetConfig.fontSize || 0.8;
 
                 function render(obj) {
-                    const specText = JSON.stringify(obj, false, indent),
```

Comment:
invalid second argument -- should be `null` or `undefined` if there is no transformation function